src/gui/folderman.cpp

```diff
@@ -1371,9 +1371,12 @@ static QString checkPathValidityRecursive(const QString &path)
     Utility::NtfsPermissionLookupRAII ntfs_perm;
 #endif
     const QFileInfo selFile(path);
+    if (QFileInfo::exists(selFile.filePath() + QStringLiteral("/.owncloudsync.log"))) {
```

Comment:
We could try to provide a different warning in this case as in theory using an old sync dir is a valid scenario...

Reply:
I would rather check on the syncjournal db file, as that is going to be removed in `Folder::wipeForRemoval`.

Reply:
Thx! ❤️